srml/system/src/lib.rs

```diff
@@ -774,6 +774,8 @@ impl<T: Trait> rstd::fmt::Debug for CheckNonce<T> {
 
 impl<T: Trait> SignedExtension for CheckNonce<T> {
 	type AccountId = T::AccountId;
+	type AdditionalSigned = ();
+	fn additional_signed(&self) -> Result<(), &'static str> { Ok(()) }
```

Comment:
probably easier to put this default impl into the trait definition and not even code it here.

Reply:
```diff
- type AdditionalSigned: Encode;
+ type AdditionalSigned: Encode + Default;

+ fn additional_signed(&self) -> Result<Self::AdditionalSigned, &'static str> { Ok(Default::default()) }
``

Reply:
This is no reason to force `AdditionalSigned` to be `Default`.